### ISSUE TITLE
Install on Edgerouter Lite and now my WAN interface is acting as a public DNS listener

### ISSUE THREAD

**Issue by jerblack** [nextdns/nextdns#453]
&lt;Describe the bug>
I noticed in the Nextdns logs that there were a ton of queries for mail servers from some random addresses in Denmark, so I assume some spam bot was using my router for DNS. Why is your daemon listening on the WAN interface? I tried to use `nextdns config` to change the listener address , but you only provide options to set new listeners, not remove them. I eventually figured out how to copy the `nextdns.conf` off the router and edit it so that it only listened on the internal IP address, but that didn't help either.  I verified from an external machine that `nslookup www.google.com <my public ip>` worked with the nextdns daemon started and failed when I stopped the daemon, so it's definitely the nextdns daemon that's handling the dns queries on the wan interface. 
After several failed attempts at trying to create a firewall rule to block incoming dns traffic from the wan interface, I ended up creating a port forwaring rule that forwarded TCP and UDP 53 to port 65534 on a nonexistent IP.  

To recap:
- your daemon listens on the WAN interface by default
- your daemon doesn't provide any warning that this is going to happen
- your daemon doesn't provide a method to meaningfully control the listener, or control  which interface  it will listen on.
- "nextdns confg" only seems to provide a means to view and add entries, but no way to remove entries without directly editing the config file.

**Context**
 - Version [e.g. 1.5.2]:  1.11.0
 - Platform [e.g. macOS, ASUS-Merlin]: EdgeRouter Lite v2.0.8-hotfix.1

**Comment by mihakralj** [comment]
Can you check if you have `setup-router` set to false and the right set of `listen` statements  in your nextdns.config?
to exclude the WAN interface, you should have:
`
setup-router false
listen 10.0.0.1:53
listen 192.168.1.1:53

**Comment by jerblack** [comment]
setup-router was set to true, so I changed it to false and ran `sudo nextdns restart`. I then removed the port forwarding rule that is blackholing external DNS requests at the moment, but I was then able to use my external publis IP address as a DNS resolver, so the change made no difference. If I don't have that port forwarding rule turned on, I am still able to use my public IP as a DNS server. I am verifying this by connecting my laptop to my phone hotsport, ensuring I got a different address, and using nslookup on windows with the port forwarding rule turned on and off. 

```
# from cmd, open nslookup in interactive mode
nslookup
server <public ip address of edgerouter>
www.google.com
```
WIth the port forwarding rule enabled, I get no response. With it disabled, I get a valid DNS response.
Here's my current nextdns.conf

```
control /var/run/nextdns.sock
use-hosts true
setup-router false
auto-activate false
log-queries false
cache-size 10MB
report-client-info true
detect-captive-portals false
listen 192.168.0.1:53
config ******
discovery-dns
hardened-privacy false
timeout 5s
cache-max-age 0s
max-ttl 5s
bogus-priv true
```

**Comment by vijvis** [comment]
I don't see this issue at all. I am running a USG 3P with NextDNS CLI installed. I tried the same test (connecting my Mac to my mobile hotspot on 4G LTE) and get the below.
NextDNS CLI version: 1.33.2

vijvis@Vijays-Mac-mini % nslookup google.com <my WAN IP>
;; connection timed out; no servers could be reached

vijvis@Vijays-Mac-mini %

**Comment by Fail-Safe** [comment]
> Why is your daemon listening on the WAN interface?

FWIW, long-time user of NextDNS here and I've never noticed the behavior you've described.

**Comment by vijvis** [comment]
Did another test today. I have now swapped over to using OPNSense firewall/router. NextDNS CLI version 1.37.7 installed on OPNSense firewall/router. Connected my Mac mini to my mobile hotspot (Telstra 4G).

vijvis@Vijays-Mac-mini ~ % nslookup
> server <Public IP of OPNSense firewall>
Default server: <Public IP of OPNSense firewall>
Address: <Public IP of OPNSense firewall>#53
> www.google.com
;; connection timed out; no servers could be reached
> exit
vijvis@Vijays-Mac-mini ~ %

NextDNS on my OPNSense firewall is not acting as a public DNS resolver. Have been using NextDNS for over two years and have never seen this issue. Looks like an issue with your setup / edge router configuration.
